### ISSUE TITLE
Correct Event API

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#3816]
- `show_map` -> `is_map_shown`. Change data type to boolean. Default `false`.
- `has_session_speakers` -> `has_sessions_speakers`.
- `ticket_include` -> `is_ticketing_enabled`. (default: true)
- `tax_allow` -> `is_tax_enabled`. 
- `pay_by_paypal` -> `is_pay_by_paypal`
- `pay_by_stripe` -> `is_pay_by_stripe_enabled`
- `pay_by_bank` -> `is_pay_by_bank_enabled`
- `pay_by_cheque` -> `is_pay_by_cheque_enabled`
- `sponsors_enabled` -> `is_sponsors_enabled`
- `email` - remove from API, model and db. Not used.
- `state` - Make all state data lower-case.

Also update db field & model for all of the above without resulting in loss of any data.

**Comment by LuD1161** [comment]
@niranjan94 
### Problem : 
- Changing data type of a column results in all values turning to **NULL** 

### Proposed Solutions
I have thought of a few methods of changing datatype without loss of data.
1. Using the bind method of **almebic** ( but you suggested that there could be even simpler method )
2. Creating a temporary column and then feeding data again to the newly created column from the temporary column. In the end deleting the temporary column.
3. Executing the raw query , cause when changing the type and to prevent loss we need to **cast** and use the statement `USING` like here.

```
    op.execute('ALTER TABLE events ALTER COLUMN show_map drop default')
    op.execute('ALTER TABLE events ALTER COLUMN show_map TYPE BOOLEAN USING (show_map::boolean)')
    op.alter_column('events', 'show_map', new_column_name='is_map_shown')
```
Currently I have tried the **1st** and **3rd** one and both are working fine.
3rd one is easier though.

Your suggestions.

**Comment by niranjan94** [comment]
I'm afk. Will reply in an hour or so.
But the key is..
First remove the default integer constraint
Then change the data type while converting the data at the same time. You
can do that in postgres.
Then set the default constrain for boolean.

**Comment by LuD1161** [comment]
@niranjan94 
Yes did that.
Thanks .

**Comment by LuD1161** [comment]
@niranjan94 
Have made all changes , check it once.

**Comment by LuD1161** [comment]
@niranjan94 
Can you check it now, please ?

**Comment by LuD1161** [comment]
Closed via [PR#3820](https://github.com/fossasia/open-event-orga-server/pull/3820)
